test/convert.js

```diff
@@ -0,0 +1,73 @@
+var fs = require('fs');
+var path = require('path');
+
+
+function convert(file, filePath) {
+  // remove version that is not being used or updated
+  delete file.version;
+
+  function traverse(obj) {
+    for (i in obj) {
+      if (!!obj[i] && typeof(obj[i])=="object") {
+        // log structures that aren't nested properly and contain a dot.
+        // Need to fix these manually or come up with a script here
+        if (i.includes(".")) {
+          console.log(i);
+        }
+        // there is __compat, but sub features follow, "support" should always be the next key
+        if (obj[i].hasOwnProperty("__compat") && !obj[i].__compat.support) {
```

Comment:
This conditional, and the comment above, confused me. AFAICS, in the old schema, `support` never follows `__compat`, so `obj[i].__compat.support` is always undefined. Perhaps the guard is to stop us processing objects that we have already converted?